Plugins/Locales/ptBR.lua

```diff
@@ -59,14 +59,14 @@ L.closeProximityDesc = [=[Fecha a janela de aproximação.
 
 Para desativala completamente para um encontro, tem que ir nas opções deste encontro e desativar a opção de 'Aproximação'.]=]
 L.colors = "Cores"
--- L.combatLog = "Automatic Combat Logging"
+L.combatLog = "Registro de Combate Automático"
 -- L.combatLogDesc = "Automatically start logging combat when a pull timer is started and end it when the encounter ends."
 L.countDefeats = "Contador de vitórias "
-L.countdownAt = "Contagem em... (segundos)" -- Needs review
-L.countdownColor = "Cor de Contagem" -- Needs review
-L.countdownTest = "Teste de Contagem" -- Needs review
--- L.countdownType = "Countdown Type"
-L.countdownVoice = "Voz de Contagem" -- Needs review
+L.countdownAt = "Contagem em... (segundos)"
+L.countdownColor = "Cor da Contagem"
+L.countdownTest = "Teste de Contagem"
+L.countdownType = "Tipo de Contagem"
```

Comment:
The full translation for "countdown" is "contagem regressiva", but in fact it is common we say only "contagem".
Furthermore, I have identified these texts within the addon interface and saw that occupy a lot of space to put the entire expression, so i think it can as it is.